roles/core/hosts_file/templates/hosts.j2

```diff
@@ -77,9 +77,7 @@
 ## Internal hosts
 
 {% set current_iceberg = j2_current_iceberg %}
-{% for host in range %}
-{{ write_host(host,hostvars[host],current_iceberg) }}
-{% endfor %}
+{% for host in range %}{{ write_host(host,hostvars[host],current_iceberg) }}{% endfor %}
```

Comment:
We loose in readability. Can you consider the change below?

```suggestion
{% for host in range -%}
{{ write_host(host,hostvars[host],current_iceberg) }}
{%- endfor %}
```

Reply:
Sure, let me test that.

Reply:
ok !